apps/api-gateway/src/common/filter/gateway-rpc-exception.filter.ts

```diff
@@ -0,0 +1,24 @@
+import { ArgumentsHost, Catch, ExceptionFilter, HttpStatus } from '@nestjs/common';
+import { HttpAdapterHost } from '@nestjs/core';
+import { CustomRpcException } from '@shared/filter/custrom-rpc-exception';
+
+@Catch()
+export class GatewayRpcExceptionFilter implements ExceptionFilter<CustomRpcException> {
+  public constructor(private readonly httpAdpterHost: HttpAdapterHost) {}
+
+  public catch(exception: CustomRpcException, host: ArgumentsHost) {
+    const { httpAdapter } = this.httpAdpterHost;
+    const ctx = host.switchToHttp();
+
+    const httpStatus = exception.status ? exception.status : HttpStatus.INTERNAL_SERVER_ERROR;
```

Comment:
요렇게 표현해줘도 괜찮을 것 같습니다~
```suggestion
    const httpStatus = exception.status ?? HttpStatus.INTERNAL_SERVER_ERROR;
```

Reply:
Nullish 병합 연산자죠!
기억을 못해서 쓸 생각을 못했네요 ㅎㅎ
[656bef3](https://github.com/GrebbMe/grabbme-be/pull/45/commits/656bef36f174ca57cb8b990ce46ab138ea046499)

위 커밋에 반영하였습니다!